src/Api/Teams/Projects.php

```diff
@@ -23,7 +25,7 @@ class Projects extends AbstractTeamsApi
     /**
      * @param array $params
      *
-     * @throws \Http\Client\Exception
```

Comment:
Please leave this alone

Reply:
I don't understand, how can I solve this?

Reply:
Change it back to what it was before?